### ISSUE TITLE
Pressing Analyze button in the absence of internet connectivity doesnot raise any error

### ISSUE THREAD

**Issue by humbletechy** [openMF/ppi-vision#48]
An error should be raised in the form of a toast or a dialogue box telling the user about issue in connectivity

**Comment by yashk2000** [comment]
Isn't this a duplicate of #46? This is what you have to check in the computer vision activity.

**Comment by humbletechy** [comment]
Nope there would be a regular check in the background but if user tries to press the analyze button even after the warning then the button should be lay unresponsive. It would not give a good impression to the user

**Comment by yashk2000** [comment]
Then why do you need to check for internet connection in ComputerVisionActivity?
Directly put a check here itself and show an error message when user presses the Analyze button.

**Comment by humbletechy** [comment]
For a better user experience at a very low cost

**Comment by yashk2000** [comment]
Won't it be more interfering if you show the same thing 2 times? Better show it only once when the internet connection is actually needed. No need to unnecessarily show the same notification twice and make things cumbersome for the user.

**Comment by humbletechy** [comment]
There won't be a notification it would be a snack bar appearing below or top of the screen so there won't be any unnecessary interference.

**Comment by yashk2000** [comment]
I understand. But I still don't get the logic of showing the same notification twice. 
Why does a user need to see it when they enter the computer vision activity?
The time connection is needed is when they want the images analyzed. Only then should the notification be shown.
What you're suggesting to do is like telling the user no internet connection in a place where a user actually doesn't even need a connection.

**Comment by yashk2000** [comment]
This issue is suitable and should be done i.e. showing a notification if the internet is absent when pressing analyze. But I suggest that you leave the other issue as that will just be useless once this one is done.

**Comment by humbletechy** [comment]
That issue is necessary because if the user sees it as it enters the activity, the user would not bother to take images, and in any case if the user mises the snackbar notification then the analyze button should not remain unresponsive
